Added new  the Look

Sd-Shivam opened the pull request:
Hi, I had make some small changes like changed the old appearance to new look 
I had add the Preview to the read me file.
I feel that its look should change now that way I changed the look

CrackSoft900 commented:
Bro.  Please tell..... When you make Your Bomb as International  SMS Bomb

On Fri, Nov 27, 2020, 11:41 Shivam-Singh <notifications@github.com> wrote:

> Hi, I had make some small changes like changed the old appearance to new
> look
> I had add the Preview to the read me file.
> I feel that its look should change now that way I changed the look
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/TheSpeedX/TBomb/pull/220
> Commit Summary
>
>    - updated
>    - appearence updated
>
> File Changes
>
>    - *M* README.md
>    <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5>
>    (6)
>    - *M* TBomb.sh
>    <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-46157afae4594249bacb163e7a2651e9295fb8b8a02beb508da559b03a559b24>
>    (36)
>    - *M* bomber.py
>    <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-556dd42d571c344638573c62de94cc75234420cb0d67e19c9fbcd1f7e83f6e7c>
>    (62)
>
> Patch Links:
>
>    - https://github.com/TheSpeedX/TBomb/pull/220.patch
>    - https://github.com/TheSpeedX/TBomb/pull/220.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TheSpeedX/TBomb/pull/220>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQO75DD6I6KNIYSDW54TBF3SR47JRANCNFSM4UEP6SAA>
> .
>

Sd-Shivam commented:
> Bro. Please tell..... When you make Your Bomb as International SMS Bomb
> […](#)
> On Fri, Nov 27, 2020, 11:41 Shivam-Singh ***@***.***> wrote: Hi, I had make some small changes like changed the old appearance to new look I had add the Preview to the read me file. I feel that its look should change now that way I changed the look ------------------------------ You can view, comment on, or merge this pull request online at: #220 Commit Summary - updated - appearence updated File Changes - *M* README.md <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5> (6) - *M* TBomb.sh <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-46157afae4594249bacb163e7a2651e9295fb8b8a02beb508da559b03a559b24> (36) - *M* bomber.py <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-556dd42d571c344638573c62de94cc75234420cb0d67e19c9fbcd1f7e83f6e7c> (62) Patch Links: - https://github.com/TheSpeedX/TBomb/pull/220.patch - https://github.com/TheSpeedX/TBomb/pull/220.diff — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#220>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQO75DD6I6KNIYSDW54TBF3SR47JRANCNFSM4UEP6SAA> .



> Bro. Please tell..... When you make Your Bomb as International SMS Bomb
> […](#)
> On Fri, Nov 27, 2020, 11:41 Shivam-Singh ***@***.***> wrote: Hi, I had make some small changes like changed the old appearance to new look I had add the Preview to the read me file. I feel that its look should change now that way I changed the look ------------------------------ You can view, comment on, or merge this pull request online at: #220 Commit Summary - updated - appearence updated File Changes - *M* README.md <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5> (6) - *M* TBomb.sh <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-46157afae4594249bacb163e7a2651e9295fb8b8a02beb508da559b03a559b24> (36) - *M* bomber.py <https://github.com/TheSpeedX/TBomb/pull/220/files#diff-556dd42d571c344638573c62de94cc75234420cb0d67e19c9fbcd1f7e83f6e7c> (62) Patch Links: - https://github.com/TheSpeedX/TBomb/pull/220.patch - https://github.com/TheSpeedX/TBomb/pull/220.diff — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#220>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQO75DD6I6KNIYSDW54TBF3SR47JRANCNFSM4UEP6SAA> .

very soon bro

0n1cOn3 reviewed (changes requested):
It doesn't look bad, but I find this not suitable for the official TBomb. You are welcome to maintain this as your fork. 

Fork of: Sd-Shivam             Made by SpeedX

Sd-Shivam commented:
> It doesn't look bad, but I find this not suitable for the official TBomb. You are welcome to maintain this as your fork.
> 
> Fork of: Sd-Shivam Made by SpeedX

ok

t0xic0der commented:
We can go ahead and close this PR, now that it is not getting merged. 

cc: @TheSpeedX
